### ISSUE TITLE
Mailing lists are no longer synchronized with the UTBM

### ISSUE THREAD

**Issue by TheoDurr** [ae-utbm/sith3#453]
Editing a mail on a mailing list is no longer adding, removing, or editing the user on the concerned mailing list.

**Comment by klmp200** [comment]
Si l'endpoint d'api ne change pas ses valeurs, faut voir avec la DSI si quelque chose a planté
Rappel, leur bot passe une fois par jour dans la nuit

**Comment by TheoDurr** [comment]
Apparemment ça fait quelque temps que ça fonctionne plus mais ça dérange pas le bureau, ils préfèrent garder le contrôle sur les listes.

**Comment by klmp200** [comment]
Ils ont le control via sympa?

**Comment by TheoDurr** [comment]
Oui, directement. Donc la feature est peut-être un peu obsolète

**Comment by klmp200** [comment]
À voir parce que ça donnait du contrôle aux responsables de Club et aux utilisateurs que pouvaient se désinscrire

**Comment by TheoDurr** [comment]
Oui, mais si elle est pas utilisée et qu'elle ne fonctionne pas on peut peut-être au moins masquer le lien.

**Comment by klmp200** [comment]
Si les gens ne sont même pas au courant que ça existe c'est normal que personne ne l'utilise
Faut en discuter un peu avec les respo de club
Surtout que t'as des boutons magiques pour remplir les mailing et je sais qu'à l'époque c'était fortement apprécié :)

**Comment by TheoDurr** [comment]
Alors avant d'en parler on va essayer de taffer dessus pour que ça marche à nouveau 😉

**Comment by klmp200** [comment]
Je viens de tester rapidement, l'endpoint est complêtement fonctionnel, les mailing sont générées correctement avec le format qu'ils nous avaient demandé quand ça a été mis en place à l'époque
Soit ils ont changé la spec, soit ils ont coupé le lien de leur côté

**Comment by TheoDurr** [comment]
Je vais les contacter.
